public/apps/configuration/panels/role-mapping/external-identities-panel.tsx

```diff
@@ -82,6 +83,7 @@ export function ExternalIdentitiesPanel(props: {
           <EuiFlexItem grow={false}>
             <EuiFormRow hasEmptyLabelSpace={arrayIndex === 0 ? true : false}>
               <EuiButton
+                id={`delete-${arrayIndex}`}
```

Comment:
No big difference. But seems in this context, we are calling this action "remove". Probably better to keep the same terminology.

Reply:
changed it to `remove-${arrayIndex}`